spec/actors/create_with_files_actor_spec.rb

```diff
@@ -22,7 +22,7 @@
       context "when uploaded_file_ids include nil" do
         let(:uploaded_file_ids) { [nil, uploaded_file1.id, nil] }
         it "will discard those nil values when attempting to find the associated UploadedFile" do
-          allow(AttachFilesToWorkJob).to receive(:perform_later)
+          expect(AttachFilesToWorkJob).to receive(:perform_later)
```

Comment:
Right, I'm saying this test *should* break if `perform_later` never gets called.  This is how you avoid cargo cult stubbing.  If you need to stub it, it should be because you expect it to be called.